### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by dialuser** [smousavi05/Unsupervised_Deep_Learning#2]
Hello, Dr. Mousavi,

I'm trying to run your Jupyter notebook.  In the notebook, the loss function to the fine-tuned model is

loss = model.train_on_batch(x=x[idx], y=[p[idx], x[idx]])

Is that correct? That resulted in the identical values for the first and last terms in the loss. Did I interpret this right?

Thanks.

**Comment by smousavi05** [comment]
this is not a loss function. loss is a variable basically storing the training errors.
